tests/Microsoft.Identity.Test.Unit/RequestsTests/OboRequestTests.cs

```diff
@@ -134,5 +138,32 @@ public async Task AcquireTokenByOboAccessTokenInCacheTestAsync()
                 Assert.AreEqual("some-access-token", result.AccessToken);
             }
         }
+
+        [TestMethod]
+        public async Task AcquireTokenByOboNullCcsTestAsync()
+        {
+            using (var httpManager = new MockHttpManager())
+            {
+                httpManager.AddInstanceDiscoveryMockHandler();
+                var extraUnexpectedHeaders = new Dictionary<string, string>() { { Constants.CcsRoutingHintHeader, "" } };
+                AddMockHandlerAadSuccess(httpManager, TestConstants.AuthorityCommonTenant, extraUnexpectedHeaders);
+
+                var cca = ConfidentialClientApplicationBuilder
+                                                         .Create(TestConstants.ClientId)
+                                                         .WithClientSecret(TestConstants.ClientSecret)
+                                                         .WithAuthority(TestConstants.AuthorityCommonTenant)
+                                                         .WithHttpManager(httpManager)
+                                                         .BuildConcrete();
+
+                UserAssertion userAssertion = new UserAssertion(TestConstants.DefaultAccessToken);
+                var result = await cca.AcquireTokenOnBehalfOf(TestConstants.s_scope, userAssertion)
+                                      .WithCcsRoutingHint("")
```

Comment:
I think you should call these methods twice for the test to be more roboust, i.e. 

AcquireToken.WithCcs("").Execute();
AcquireToken.WithCcs("", "").Execute();